Sources/Units/Unit/DefaultUnits.swift

```diff
@@ -1055,4 +1055,13 @@ enum DefaultUnits {
         dimension: [.Length: 3],
         coefficient: 0.00025
     )
+
+    // MARK: Unitless
+
+    static let percent = try! DefinedUnit(
```

Comment:
Please add this to the `Registry.swift` file as well

Reply:
Oh will do.